Products/ZenRRD/zencommand.py

```diff
@@ -384,6 +384,14 @@ def __init__(self,
         self.executed = 0
         self._lastErrorMsg = ''
 
+        self.manage_ip_event = {
+            'eventClass': Cmd_Fail,
+            'component': 'command',
+            'device': self._devId,
+            'summary': 'Unknown host',
+            'component' : COLLECTOR_NAME,
```

Comment:
The summary is vague, it should be more specific. Maybe something like "IP address not set, collection will be attempted with host name"